Sources/Util/MissingUnicode.swift

```diff
@@ -0,0 +1,983 @@
+// MARK: - Missing stdlib API
+
+extension Unicode {
+  public enum Script: String {
```

Comment:
We may want to separate out the in-source spellings as UTS#18 has fuzzy matching rules and aliases for properties (e.g ignoring case, whitespace, and `_`, and e.g `Latn` being an alias for `Latin`), but that can be done when we implement the lexing/parsing for them.

Reply:
CC @natecook1000, resident expert on character classes and properties.